### PR TITLE
Update to latest cf

### PR DESCRIPTION
**Pull request by xingweitian**

### CODE REVIEW

File: dependency-setup.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ else
     (cd $JSR308 && git clone -b $BRANCH --depth 1 https://github.com/"$REPO_SITE"/checker-framework-inference.git)
 fi
 
-(cd $JSR308/checker-framework-inference && ./.travis-build-without-test.sh)
+(cd $JSR308/checker-framework-inference && ./.travis-build-without-test.sh && ./gradlew testLibJar)
```
Comment (wmdietl) [inline dependency-setup.sh]: Should the `./gradlew testLibJar` part go into `.travis-build-without-test.sh`?

File: worked-benchmarks.yml [hunk 2]
```diff
@@ -14,10 +14,10 @@ projects:
     build: mvn install
     clean: mvn clean
 
-  jReactPhysics3D:
-    giturl: https://github.com/opprop-benchmarks/jReactPhysics3D.git
-    build: mvn install
-    clean: mvn clean
+#  jReactPhysics3D:
```
Comment (wmdietl) [inline worked-benchmarks.yml]: Could you add a comment and/or issue reference here to make clear why this is commented out?
Or remove the project here and add that comment in the other .yml file?

File: dependency-setup.sh (hunk 1 above)
Comment (xingweitian) [reply dependency-setup.sh]: Thanks for pointing out! I will do it.

File: worked-benchmarks.yml (hunk 2 above)
Comment (xingweitian) [reply worked-benchmarks.yml]: Get it.

**Pull request merged**